### PR TITLE
replace input with wk-search, add clear input function

### PR DESCRIPTION
**Pull request by Olga1984**

### CODE REVIEW

File: client/src/shared/searchbar/searchbarController.js [hunk 1]
```diff
@@ -9,6 +9,9 @@ sharedModule.controller('SearchbarController', ['$state', function SearchbarCont
             query: this.input,
         });
     };
+    this.clearSearchInput=function(){
+        this.input = null;
```
Comment (gshokanov) [inline client/src/shared/searchbar/searchbarController.js]: Please change null to empty string. Changing variable types can slow down performance dramatically

**Review by gshokanov — CHANGES_REQUESTED** [review]

File: client/src/shared/searchbar/searchbarController.js (hunk 1 above)
Comment (Olga1984) [reply client/src/shared/searchbar/searchbarController.js]: Updated. Thanks!

File: client/src/shared/searchbar/searchbarController.js [hunk 2]
```diff
@@ -9,6 +9,9 @@ sharedModule.controller('SearchbarController', ['$state', function SearchbarCont
             query: this.input,
         });
     };
+    this.clearSearchInput=function(){
```
Comment (gshokanov) [inline client/src/shared/searchbar/searchbarController.js]: Please change this to arrow function

**Review by gshokanov — CHANGES_REQUESTED** [review]

File: client/src/shared/searchbar/searchbarController.js (hunk 2 above)
Comment (Olga1984) [reply client/src/shared/searchbar/searchbarController.js]: Updated. Thanks!

**Review by gshokanov — APPROVED** [review]

**Pull request merged**